packages/@okta/vuepress-site/docs/reference/api/rulesets/index.md

```diff
@@ -0,0 +1,979 @@
+              }
+           ],
+           "reauthenticateIn": "PT1M"
+       }
+    ]
+}' "https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules/{ruleId}"
+```
+
+##### Response Types
+
+HTTP 200:
+[Rule object](#rule-object)
+
+### Activate A Rule
+
+<ApiOperation method="post" url="/api/v1/policies/${rulesetId}/rules/${ruleId}/lifecycle/activate" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#ruleset-object) is required.
+
+The rule ID described in the [Rule object](#rule-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules/{ruleId}/lifecycle/activate"
+```
+
+##### Response Types
+
+HTTP 204:
+*No content*
+
+### Deactivate A Rule
+
+<ApiOperation method="post" url="/api/v1/policies/${rulesetId}/rules/${ruleId}/lifecycle/deactivate" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+The rule ID described in the [Rule object](#rule-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/rules/{ruleId}/lifecycle/deactivate"
+```
+
+##### Response Types
+
+HTTP 204:
+*No content*
+
+## Resource Mappings
+
+### Get Resource Mappings in a Rule Set
+
+<ApiOperation method="get" url="/api/v1/policies/${rulesetId}/mappings" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}/mappings"
+```
+
+##### Response Types
+
+HTTP 200:
+[Mapping object](#mappings-object)
+
+### Create a Resource Mapping
+
+<ApiOperation method="post" url="/api/v1/policies/${rulesetId}/mappings" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "resourceType": "APP",
+  "resourceId": "0oa10510BvfrKTfzD0g4"
+}' "https://${yourOktaDomain}/api/v1/policies/{rulesetId}/mappings"
+```
+
+##### Response Types
+
+HTTP 200:
+[Mapping object](#mapping-object)
+
+### Delete a Resource Mapping
+
+<ApiOperation method="delete" url="/api/v1/policies/${rulesetId}/mappings/${mappingId}" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+The mapping ID described in the [Mapping object](#mapping-object) is required.
```

Comment:
This object doesn't seem to be documented anywhere.

Reply:
What information do you need? Are you looking for something from me? 👀

Reply:
Good catch, here's what a Resource Mapping object looks like. It literally just had `id` and `_links`.

```json
{
        "id": "rsm30qv3iirGzUyaM0g7",
        "_links": {
            "application": {
                "href": "https://your-org.okta.com/api/v1/apps/0oa1z79p3ozTgE5cw0g7",
                "hints": {
                    "allow": [
                        "GET",
                        "PUT",
                        "DELETE"
                    ]
                }
            },
            "self": {
                "href": "https://your-org.okta.com/api/v1/policies/rst30qv3igD5OpiL50g7/mappings/rsm30qv3iirGzUyaM0g7",
                "hints": {
                    "allow": [
                        "GET",
                        "DELETE"
                    ]
                }
            },
            "policy": {
                "href": "https://your-org.okta.com/api/v1/policies/rst30qv3igD5OpiL50g7",
                "hints": {
                    "allow": [
                        "GET",
                        "PUT",
                        "DELETE"
                    ]
                }
            }
        }
    }
```